src/components/Filters/AggregationBranch.jsx

```diff
@@ -85,7 +91,7 @@ export class AggregationBranch extends React.Component {
                  disabled={item.disabled}
                  checked={checkedState === CHECKED}
                  className="flex-fixed a-checkbox"
-                 id={id}
+                 id={ sanitizeHtmlId( id ) }
```

Comment:
You sanitize it twice?  It better be clean now!

Reply:
ah whoops. that was an accident